packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -282,9 +282,29 @@ describe('ListView', function () {
     });
   });
 
-  it('should display loading affordance', function () {
+  it('should display loading affordance with proper height (isLoading)', function () {
     let {getByRole} = render(<ListView aria-label="List" loadingState="loading">{[]}</ListView>);
-    expect(getByRole('progressbar')).toBeTruthy();
+    let progressbar = getByRole('progressbar');
+    expect(progressbar).toBeTruthy();
+    expect(progressbar.parentNode.parentNode.parentNode.style.height).toBe('1000px');
+  });
+
+  it('should display loading affordance with proper height (isLoadingMore)', function () {
+    let items = [
+      {key: 'foo', label: 'Foo'},
+      {key: 'bar', label: 'Bar'},
+      {key: 'baz', label: 'Baz'}
+    ];
+    let {getByRole} = render(
+      <ListView items={items} aria-label="List" loadingState="loadingMore">
+        {item =>
+          <Item textValue={item.key}>{item.label}</Item>
+        }
+      </ListView>
+    );
+    let progressbar = getByRole('progressbar');
+    expect(progressbar).toBeTruthy();
+    expect(progressbar.parentNode.parentNode.parentNode.style.height).toBe('40px');
```

Comment:
three parentNode requests annoy me, but I'm figuring there isn't another way to get that.
Should this test use a CSS variable instead of hard coding 40px?
Should we look at spacious, compact, scale, etc.?

Reply:
I guess I can change it to find the last row and check that the progress bar is within instead. The height definition comes from hardcoded values within ListView so I can't use a CSS var. I would prefer that those extra tests come as followup to this PR (or even better: followup Chromatic stories since those affect the visual appearance of the row)